docker/ray-deps/Dockerfile

```diff
@@ -3,11 +3,15 @@ FROM rayproject/base-deps:nightly"$GPU"
 # If this arg is not "autoscaler" then no autoscaler requirements will be included
 ARG AUTOSCALER="autoscaler"
 ARG WHEEL_PATH
+ARG FIND_LINKS_PATH
```

Comment:
Could you have these default to `ARG FIND_LINKS_PATH=".whl"`
(for all Dockerfiles)

Reply:
OK